joomla/joomla-cms
# com_installer update view: searchtools ordering and some improvements

#### Description

This PR adds, for consistency, as it exists in so many other views in the backoffice, the order select box of the search tools to com_installer update view.

Also:
- rearrange columns (install type moves to end)
- corrects bug where the folder filter has not selecting the search button
- corrects bug where by searching or selecting a filter the search tools resets when it should show the selected option
- correct visual on mobile, tablet and desktop
- other small graphical issues.
###### Before PR

![image](https://cloud.githubusercontent.com/assets/9630530/13029883/3ffa06fc-d290-11e5-8036-5314df209de5.png)
###### After PR

![image](https://cloud.githubusercontent.com/assets/9630530/13029941/0fd0b46a-d292-11e5-8bd5-6e0d3e563ded.png)
#### How to test
1. Install latest staging and apply this patch
2. Install some old version extensions (which already have updates)
3. Go to Extensions -> Manage -> Update
4. Click "Find Updates" (check the updates for the extensions you installed are there)
5. Check the search, ordering select box and the filter are working fine.
6. Test desktop, tablet and mobile view (ex: reduce browser width).
7. Update your extensions and check if are ok.
